hapi/hedera-protobufs/services/address_book_service.proto

```diff
@@ -91,18 +91,20 @@ import "transaction.proto";
  *       - If the transaction changes the value of the "node account" the
  *         node operator MUST _also_ sign this transaction with the active `key`
  *         for the account to be assigned as the new "node account".
```

Comment:
This block has a duplicate below, and can be removed from here.

Reply:
Removed.